packages/trader/src/App/Constants/routes-config.js

```diff
@@ -19,8 +19,11 @@ import Trade from 'Modules/Trading';
 
 const ContractDetails = lazy(() => import(/* webpackChunkName: "contract" */  'Modules/Contract'));
 
+// MT5 Routes
+const MT5 = lazy(() => import(/* webpackChunkName: "mt5" */            'Modules/MT5'));
```

Comment:
Can we use `webpackPrefetch: true` for this chunk? since the user doesn't need it on page load, we can store it in cache for later use.

Reply:
@reza-fs I agree.